apps/user/services/authentication.go

```diff
@@ -198,6 +193,35 @@ func (service AuthenticationService) Recover(c *gin.Context) (
 	return http.StatusOK, nil
 }
 
+// UpdatePassword ...
+func (service AuthenticationService) UpdatePassword(userPasswordData userSerializers.Recover) (
+	status int,
+	err error) {
+
+	encryptedPassword := EncryptPassword(userPasswordData.Password)
+	tx := service.DB.Begin()
+	user := userModels.User{}
+
+	err = tx.Model(&userModels.User{}).Where("email = ?", userPasswordData.Email).Update("password", encryptedPassword).Scan(&user).Error
+	if err != nil {
+		tx.Rollback()
+		return http.StatusInternalServerError, err
+	}
+
+	err = tx.Where("user_id = ?", user.ID).Delete(&userModels.OTP{}).Error
```

Comment:
Please add proper comments here and other places.

Reply:
done.